document_page_approval_levels/views/document_page_view.xml

```diff
@@ -6,6 +6,38 @@
             <field name="model">document.page</field>
             <field name="inherit_id" ref="document_page.view_wiki_form"/>
             <field name="arch" type="xml">
+                <xpath expr="//field[@name='name']" position="after">
+                    <div class="oe_button_box oe_right">
+                        <field name="content_can_print" invisible="1"/>
+                        <button name="%(lodi_document.lodi_document_page_report)d"
```

Comment:
To avoid this and the missing dependency (document_page_approval_levels are going to need depends on lodi_document or this will fails), this part of the code must be directly on lodi_document module ok @LuisAlejandro